sdk/eventhub/tests.yml

```diff
@@ -1,7 +1,7 @@
 trigger: none
 
 extends:
-  template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+  template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests-multiple.yml
```

Comment:
@danieljurek:  This was what the prototype PR was using.  Please let me know if it's not something that we should opt-into this just yet,

Reply:
This should just be `archetype-sdk-tests.yml` ... The `-multiple` will cause live tests to fail.

Reply:
Thanks.  I noticed that it didn't exist yet and wasn't sure if it was pending, something that I should have included here, or something that I should ignore.

For now, I'll take the latter tact and ignore it.